authors/apps/bookmarks/views.py

```diff
@@ -0,0 +1,178 @@
+from rest_framework import status
+from rest_framework.generics import RetrieveAPIView, ListAPIView
+from rest_framework.permissions import (
+    IsAuthenticated, IsAuthenticatedOrReadOnly)
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from .permissions import IsOwner
+from authors.apps.authentication.models import User
+from authors.apps.articles.models import Articles
+from .serializers import BookmarkSerializer
+from .models import Bookmark
+from authors.apps.authentication.permissions import IsVerifiedUser
+from .response_messages import BOOKMARK_MSGS
+
```

Comment:
Check for unused imports, eg. line 2, line 8, and omit them

Reply:
Imports have been updated